src/sentry/api/endpoints/project_commits.py

```diff
@@ -22,12 +22,18 @@ def get(self, request: Request, project) -> Response:
                                           commit belongs to.
         :pparam string project_slug: the slug of the project to list the
                                      commits of.
+        :qparam string query: this parameter can be used to create a
```

Comment:
Hmm, `query` usage here..

I searched around and, generally, the `query` queryparam seems to follow a common convention - one that your usage doesn't follow.

Maybe someone from backend-api can chime in on the proper syntax for this.

Reply:
I used `ProjectReleasesEndpoint` and the usage of `query` there as a model since the use case is the same.